src/modules/quoiFeur/quoiFeur.helpers.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  ChannelType,
+  type ChatInputCommandInteraction,
+  Client,
+  Guild,
+  type Message,
+  Role,
+} from 'discord.js';
+
+import { cache } from '../../core/cache';
+import { removeEmoji, removePunctuation } from '../../helpers/regex.helper';
+
+const ONE_MINUTE = 1 * 60 * 1000;
+const MUTED_ON_COUBEH = 'Muted on Coubeh';
+
+const quoiDetectorRegex = /\bquoi\s*$/i;
+const endWithQuoi = (text: string) => quoiDetectorRegex.test(removeEmoji(removePunctuation(text)));
+
+const reactWithFeur = async (message: Message) => {
+  await message.react('🇫');
+  await message.react('🇪');
+  await message.react('🇺');
+  await message.react('🇷');
+};
+
+const reactWithCoubeh = async (message: Message) => {
+  await message.react('🇨');
+  await message.react('🇴');
+  await message.react('🇺');
+  await message.react('🇧');
+  await message.react('🇪');
+  await message.react('🇭');
+  await message.react('🔇');
+
+  const mutedRole = message.guild?.roles.cache.find((r) => r.name === MUTED_ON_COUBEH);
+
+  if (!mutedRole?.id) return;
+
+  await message.member?.roles.add(mutedRole.id);
+
+  setTimeout(() => {
+    message.member?.roles.remove(mutedRole.id).catch(console.error);
+  }, ONE_MINUTE * 5);
+};
+
+export const reactOnEndWithQuoi = async (message: Message) => {
+  const channelIds = await cache.get('quoiFeurChannels', []);
+  const channelHasGame = channelIds.find((channelId) => channelId === message.channelId);
+  if (!channelHasGame) return;
+
+  if (!endWithQuoi(message.content)) return;
+
+  const probability = 1 / 20;
+
+  try {
+    Math.random() <= probability ? await reactWithCoubeh(message) : await reactWithFeur(message);
+  } catch (error) {
+    console.error(error);
+  }
```

Comment:
This is redundant, as when a feature crash, it already prints out the error.
```suggestion
  Math.random() <= probability ? await reactWithCoubeh(message) : await reactWithFeur(message);
```